assets/js/modules/ads/components/notifications/AdsModuleSetupCTABanner.js

```diff
@@ -162,49 +153,58 @@ export default function AdsModuleSetupCTABanner( { id, Notification } ) {
 
 	return (
 		<Notification>
-			<NotificationWithSVG
-				id={ id }
+			<SetupCTA
+				notificationID={ id }
 				title={ __(
 					'Get better quality leads and enhance conversions with Ads',
 					'google-site-kit'
 				) }
 				description={
-					<Description
-						text={ __(
-							'Help drive sales, leads, or site traffic by getting your business in front of people who are actively searching Google for products or services you offer.',
-							'google-site-kit'
+					<Fragment>
+						{ createInterpolateElement(
+							__(
+								'Help drive sales, leads, or site traffic by getting your business in front of people who are actively searching Google for products or services you offer. <a />',
+								'google-site-kit'
+							),
+							{
+								a: (
+									<LearnMoreLink
+										id={ id }
+										label={ __(
+											'Learn more',
+											'google-site-kit'
+										) }
+										url={ learnMoreURL }
+									/>
```

Comment:
Learn more link is uesd here instead of the prop so that the learn more link can show above the `AdBlockerWarning`.